.github/issue-close-app.yml

```diff
@@ -0,0 +1,30 @@
+# Comment that will be sent if an issue is judged to be closed.
+comment: "This issue is closed because it does not meet our issue template. Please follow the initial template that appears when you open a new issue. Your submission should at a minimum include at least the words 'NAME' if submitting an email request, 'Description' for a bug, or 'Feature' for a feature request."
+issueConfigs:
+# There can be several configs for different kind of issues.
+- content:
+# bug report
+  - "Description"
+- content:
+# email add request 
+  - "NAME"
+- content:
+# feature 
+  - "Feature"
+# The issue is judged to be legal if it includes all keywords from any of these two configs.
```

Comment:
just leave out the "two". It's wrong now and it might just become more wrong later.